models/protocols/SendGridProtocol.cfc

```diff
@@ -32,6 +32,10 @@ component extends="cbmailservices.models.AbstractProtocol" {
         
         body[ "subject" ] = mail.subject;
         
+        if( structKeyExists( mail, "additionalInfo" ) && isStruct( mail.additionalInfo ) && structKeyExists( mail.additionalInfo, "categories" ) ){
+            body[ "categories" ] = listToArray( mail.additionalInfo.categories );            
```

Comment:
Can a user set `mail.additionalInfo.categories` to an array instead of a list?  Maybe we should do a quick check for that.

```
if ( ! isArray( mail.additionalInfo.categories ) ) {
    mail.additionalInfo.categories = listToArray( mail.additionalInfo.categories );
}
body[ "categories" ] = mail.additionalInfo.categories;
```